docs/openapi/openapi.yaml

```diff
@@ -302,13 +357,43 @@ components:
       allOf:
         - $ref: "#/components/schemas/Person"
       type: object
+      required:
+        - dateOfBirth
+        - email
+        - status
       properties:
+        dateOfBirth:
+          type: string
+          format: date
+        email:
+          type: string
+          x-faker: internet.email
         status:
           type: string
           enum:
             - active
             - replacement
             - removed
+    CertificateProvider:
+      allOf:
+        - $ref: "#/components/schemas/Person"
+      type: object
+      required:
+        - email
+        - carryOutBy
```

Comment:
Been wondering if carryOutBy might be better as ApplicationChannel.

Reply:
they aren't applying, so maybe `channel` if we wanted to change it?